mkdocs.yml

```diff
@@ -52,6 +52,8 @@ nav:
         - Level Editing:
           - Sonic GLvl: guides/hedgehog-engine/blueblur/levels/glvl.md
       - Sonic Frontiers:
+       - Cyloop Color:
+          - Replacing the Color: guides/hedgehog-engine/rangers/Cyloop/color.md
```

Comment:
Folder name is in upper case, please rename "Cyloop" to "cyloop" to correctly match the directory, otherwise the page might not show up